### ISSUE TITLE
Bind phablet home to anbox userspace

### ISSUE THREAD

**Issue by nfsprodriver** [ubports/anbox#11]

**Comment by mariogrip** [comment]
I'm not sure if this is something everyone wants, I'm totally up for having this as an option though.

A good example is that someone maybe want to keep android and ubuntu touch separate for privacy and security reasons.

**Comment by nfsprodriver** [comment]
Good point to offer this as option. Anyway, here's a fancy start to implement this https://github.com/ubports/anbox/commit/699458564cbd13e0c040c7b43fbd4d4be849c404

**Comment by ZachIndigo** [comment]
Very much pro this, it'd be a nice convenience to have as an option.
